package.json

```diff
@@ -4,37 +4,35 @@
   "description": "Application to connect Hemi Network to External Services",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "type": "module",
   "private": true,
   "scripts": {
-    "give-points": "bun ./src/presentation/cli/GivePoints.ts",
     "lint": "eslint",
     "lint:fix": "eslint --fix",
     "test": "vitest run",
     "test:cov": "vitest run --coverage",
-    "test:watch": "vitest watch"
+    "test:watch": "vitest watch",
+    "build": "tsc -p ./tsconfig.build.json",
+    "collect-metrics": "node dist/presentation/cli/CollectMetrics.js",
+    "give-points": "node ./src/presentation/cli/GivePoints.ts"
   },
-  "author": "Thomas Alvarenga (thomas@bloq.com)",
+  "author": "Thomas Alvarenga (thomas@hemi.xyz)",
   "license": "MIT",
   "dependencies": {
-    "hemi-viem": "^1.1.0",
-    "typescript": "^5.6.3",
-    "viem": "^2.17.3",
-    "web3": "^4.14.0"
+    "hemi-viem": "1.7.0",
+    "typescript": "5.6.3",
+    "uuid": "11.0.3",
+    "viem": "2.17.3"
   },
   "devDependencies": {
-    "@types/node": "^18.7.15",
-    "@typescript-eslint/eslint-plugin": "^5.0.0",
-    "@vitest/coverage-c8": "^0.23.1",
-    "bun": "^1.1.34",
-    "eslint": "^8.0.1",
-    "eslint-config-standard-with-typescript": "^22.0.0",
-    "eslint-plugin-import": "^2.25.2",
-    "eslint-plugin-n": "^15.0.0",
-    "eslint-plugin-promise": "^6.0.0",
-    "eslint-plugin-sonarjs": "^0.15.0",
-    "ts-node": "^10.9.1",
-    "tsx": "^4.7.1",
-    "vitest": "^0.23.1"
+    "@types/node": "18.19.62",
+    "@typescript-eslint/eslint-plugin": "5.62.0",
+    "@vitest/coverage-c8": "0.23.4",
+    "eslint": "8.57.1",
+    "eslint-config-standard-with-typescript": "22.0.0",
+    "eslint-plugin-import": "2.31.0",
+    "eslint-plugin-n": "15.7.0",
+    "eslint-plugin-promise": "6.6.0",
+    "eslint-plugin-sonarjs": "0.15.0",
```

Comment:
suggestion for another PR: use [eslint-config-bloq](https://github.com/bloq/eslint-config-bloq/) - available in [npm](https://www.npmjs.com/package/eslint-config-bloq)